assets/js/base/components/chip/style.scss

```diff
@@ -0,0 +1,40 @@
+.wc-block-components-chip {
+	display: inline-block;
+	background: $core-grey-light-500;
+	padding: 0.365em 0.5em;
+	margin: 0 0.365em 0.365em 0;
+	color: $core-grey-dark-800;
+	border-radius: 0;
+	line-height: 1em;
+
+	&.wc-block-components-chip--radius-small {
+		border-radius: 3px;
+	}
+	&.wc-block-components-chip--radius-medium {
+		border-radius: 0.433em;
+	}
+	&.wc-block-components-chip--radius-large {
+		border-radius: 0.865em;
+		padding: 0.365em 0.75em;
+	}
+
+	.wc-block-components-chip__remove {
+		background: transparent;
+		border: 0;
+		padding: 0;
+		appearance: none;
+		float: none;
+		vertical-align: middle;
+		margin: 0 0 0 0.5em;
+		font-size: 12px;
+
+		&:hover,
+		&:focus {
+			color: #d94f4f;
```

Comment:
Does this colour have a semantic meaning? Using a variation of red feels a little too opinionated / inflexible.

Could also have a hover affordance showing the click target as a grey circle, then themes could tweak the colours easily.

Reply:
This is a red from our mixins. There is a pattern in Gutenberg (https://developer.wordpress.org/block-editor/components/button/) for destructive things showing in red. I'm not sure how a color overridable with CSS is any more or less flexible that your suggestion of a grey circle?

With the absence of designs for hover states, I don't think this should block the PR cc @nerrad